src/cloud/project/manage-disk-space.md

```diff
@@ -9,6 +9,9 @@ You can find the total storage capacity for your Cloud project in your {{site.da
 
 You can check disk space usage in each environment using the _disk free_ command, which reports the amount of disk space used by the file system. You must use SSH to log in to a remote environment.
 
+{:.bs-callout-info}
+The _df -h_ command is only for dedicated clusters i.e. use it on Staging and Production environments of Magento Cloud Pro projects. On the Integration environments and for Magento Cloud Starter projects, use _magento-cloud db:size_
```

Comment:
**Staging and Production environments**: to check the disk space on these clusters "df -h" can be used. Sample output below. /data/exports/ mount shows the disk space for media and /data/mysql/ mount shows disk space for database

```
sngst3u29h5gf@i-04422e3b24f2ef115:~$ df -h
Filesystem                                    Size  Used Avail Use% Mounted on
udev                                           16G     0   16G   0% /dev
tmpfs                                         3.2G  9.1M  3.2G   1% /run
/dev/xvda1                                     59G  8.9G   48G  16% /
tmpfs                                          16G   36K   16G   1% /dev/shm
tmpfs                                         5.0M     0  5.0M   0% /run/lock
tmpfs                                          16G     0   16G   0% /sys/fs/cgroup
/dev/xvdj                                     9.8G  2.3G  7.6G  23% /data/mysql
/dev/xvdi                                     9.8G  491M  9.3G   5% /data/exports
192.168.5.5:/shared                           9.8G  591M  9.3G   6% /mnt/shared
/dev/loop0                                     91M   91M     0 100% /app/sngst3u29h5gf
192.168.5.5:/shared/sngst3u29h5gf/var         9.8G  591M  9.3G   6% /app/sngst3u29h5gf/var
192.168.5.5:/shared/sngst3u29h5gf/app/etc     9.8G  591M  9.3G   6% /app/sngst3u29h5gf/app/etc
192.168.5.5:/shared/sngst3u29h5gf/pub/media   9.8G  591M  9.3G   6% /app/sngst3u29h5gf/pub/media
192.168.5.5:/shared/sngst3u29h5gf/pub/static  9.8G  591M  9.3G   6% /app/sngst3u29h5gf/pub/static
```
**Integration environment and Starter projects**: Use "magento-cloud mount:size"

Sample response:

```
Checking disk usage for all mounts on <project>-<branch>-mymagento@ssh.us.magento.cloud...

+------------+-----------+---------+-----------+-----------+--------+
| Mount(s)   | Size(s)   | Disk    | Used      | Available | % Used |
+------------+-----------+---------+-----------+-----------+--------+
| app/etc    | 184 KiB   | 1.9 GiB | 481.3 MiB | 1.4 GiB   | 24.7%  |
| pub/media  | 128 KiB   |         |           |           |        |
| pub/static | 158.2 MiB |         |           |           |        |
| var        | 316.7 MiB |         |           |           |        |
+------------+-----------+---------+-----------+-----------+--------+
```